examples/docs/src/layouts/Main.astro

```diff
@@ -23,12 +23,10 @@ const githubEditUrl = `https://github.com/USER/REPO/blob/main/${currentFile}`
     <title>{content.title}</title>
     
     <link rel="stylesheet" href="/index.css" />
-    <link rel="stylesheet" href="/theme.css" />
-    <link rel="stylesheet" href="/code.css" />
```

Comment:
Hmm, this undid my PR https://github.com/snowpackjs/astro/pull/1062 and broke the example project